app/javascript/components/public_health/CustomExport.js

```diff
@@ -208,7 +220,7 @@ class CustomExport extends React.Component {
                   {this.state.selected_records === 'current' && (
                     <div className="px-1 pb-1">
                       <Badge variant="secondary" className="mr-1">
-                        {this.props.patient_query.workflow === 'isolation' ? 'Isolation' : 'Exposure'} - {this.props.tabs[this.props.patient_query.tab]?.label}
+                        {`${this.firstLetterToUpper(this.props.patient_query.workflow)} - ${this.props.tabs[this.props.patient_query.tab]?.label}`}
```

Comment:
you can actually use `_.capitalize()` here

Reply:
Thanks, didn't know lodash had that https://github.com/SaraAlert/SaraAlert/pull/937/commits/5b32a8a7009cc9386a71b5e1ff1e445041b2e34e

Reply:
neither did I until I checked... lodash has all sorts of useful functions.  I feel like every time I'm like "I wonder if they have this" they always do